Dockerfile-dev

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.16-buster
```

Comment:
:bulb: We can use the convention `dev.dockerfile` here for the name of this file.